### ISSUE TITLE
Sometimes when deleting a service, task db is closed twice

### ISSUE THREAD

**Issue by john-kheir** [zero-os/0-robot#156]
**Got this error while trying to run a blueprint to create a vm on a zeroos node**
## Error
```
Exception ignored in: <bound method TaskList.__del__ of <zerorobot.task.task_list.TaskList object at 0x7f626a7d9a58>>
Traceback (most recent call last):
  File "/root/0-robot/zerorobot/task/task_list.py", line 43, in __del__
    self._done.close()
  File "/root/0-robot/zerorobot/task/storage/sqlite.py", line 114, in close
    self._cursor.close()
sqlite3.ProgrammingError: Cannot operate on a closed database.
```

**Comment by john-kheir** [comment]
All logs
```
[Wed04 09:14] - vm.py             :33  :service-2c440085-f1f3-4bdc-b37b-0f2a31a1cbfb - INFO    
* Creating kvm vm1
[Wed04 09:14] - node.py           :64  :service-f7dd4b94-2fef-4a68-8217-d1a70c40ea70 - INFO     - Monitoring node node1
[Wed04 09:14] - node.py           :114 :service-f7dd4b94-2fef-4a68-8217-d1a70c40ea70 - INFO     - storage pool 54a744025271_fscache doesn't exist on node 54a744025271
Exception ignored in: <bound method TaskList.__del__ of <zerorobot.task.task_list.TaskList object at 0x7f626a7d9a58>>
Traceback (most recent call last):
  File "/root/0-robot/zerorobot/task/task_list.py", line 43, in __del__
    self._done.close()
  File "/root/0-robot/zerorobot/task/storage/sqlite.py", line 114, in close
    self._cursor.close()
sqlite3.ProgrammingError: Cannot operate on a closed database.
[Wed04 09:14] - node.py           :64  :service-f7dd4b94-2fef-4a68-8217-d1a70c40ea70 - INFO     - Monitoring node node1
[Wed04 09:14] - node.py           :114 :service-f7dd4b94-2fef-4a68-8217-d1a70c40ea70 - INFO     - storage pool 54a744025271_fscache doesn't exist on node 54a744025271
[Wed04 09:15] - node.py           :64  :service-f7dd4b94-2fef-4a68-8217-d1a70c40ea70 - INFO     - Monitoring node node1
[Wed04 09:15] - node.py           :114 :service-f7dd4b94-2fef-4a68-8217-d1a70c40ea70 - INFO     - storage pool 54a744025271_fscache doesn't exist on node 54a744025271
[Wed04 09:15] - node.py           :64  :service-f7dd4b94-2fef-4a68-8217-d1a70c40ea70 - INFO     - Monitoring node node1
[Wed04 09:15] - node.py           :114 :service-f7dd4b94-2fef-4a68-8217-d1a70c40ea70 - INFO     - storage pool 54a744025271_fscache doesn't exist on node 54a744025271
[Wed04 09:16] - base.py           :201 :service-2c440085-f1f3-4bdc-b37b-0f2a31a1cbfb - ERROR    - error executing action install:
Traceback (most recent call last):
  File "/root/0-robot/zerorobot/task/task.py", line 78, in execute
    self._result = self.func()
  File "/opt/code/github/zero-os/0-templates/templates/vm/vm.py", line 49, in install
    self.data['uuid'] = self.node_sal.hypervisor.create(**args).uuid
  File "/opt/code/github/jumpscale/lib9/JumpScale9Lib/clients/zero_os/sal/Hypervisor.py", line 22, in create
    tags=tags)
  File "/opt/code/github/jumpscale/lib9/JumpScale9Lib/clients/zero_os/KvmManager.py", line 189, in create
    return self._client.json('kvm.create', args, tags=tags)
  File "/opt/code/github/jumpscale/lib9/JumpScale9Lib/clients/zero_os/ContainerManager.py", line 128, in json
    result = self.sync(command, arguments, tags=tags, id=id)
  File "/opt/code/github/jumpscale/lib9/JumpScale9Lib/clients/zero_os/ContainerManager.py", line 114, in sync
    result = response.get()
  File "/opt/code/github/jumpscale/lib9/JumpScale9Lib/clients/zero_os/Response.py", line 268, in get
    raise TimeoutError()
TimeoutError


```

**Comment by zaibon** [comment]
fixed in 48ceb2e78e0283dc380a3c45e5c6cfcdea8b91fe
